### ISSUE TITLE
Redundant 'navigation' text in aria-label for Docs pages navigation

### ISSUE THREAD

**Issue by wwu-stephanie** [facebook/docusaurus#8834]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#issues).

### Prerequisites

- [X] I'm using the latest version of Docusaurus.
- [X] I have tried the `npm run clear` or `yarn clear` command.
- [X] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages.
- [X] I have tried creating a repro with https://new.docusaurus.io.
- [X] I have read the console error message carefully (if applicable).

### Description

On the pagination nav, the aria-label attribute is set as "Docs pages navigation." The word navigation is redundant, since it appears inside a \<nav\> element. This results in it being read out loud like "Docs pages **navigation navigation** region" or something similar. 

Just for context: other navigation elements on the page seem to take this into account already, for example the "Pattern navigation" aria-label is just "Pattern"

### Steps to reproduce

1. Start JAWS
2. Navigate to https://docusaurus.io/docs/next
3. Tab to the "Docs pages navigation navigation region"

Alternatively:

1. Navigate to https://docusaurus.io/docs/next
2. Scroll to bottom of page
3. Inspect "Next Getting Started" page element
4. See aria-label on `<nav class="pagination-nav docusaurus-mt-lg" aria-label="Docs pages navigation">`

### Expected behavior

The region should announce as "Docs pages navigation region"

### Actual behavior

"Docs pages navigation **navigation** region" (e.g. navigation is repeated)

### Your environment

- Public source code: https://github.com/facebook/docusaurus
- Public site URL: https://docusaurus.io/
- Docusaurus version used: 2.4.0
- Environment name and version: Chrome 111 and JAWS

### Self-service

- [X] I'd be willing to fix this bug myself.
